doc/project/checklist.md

```diff
@@ -28,15 +32,19 @@ Unit `.sv` exists, meet comportability requirements.
 ### IP_INSTANTIABLE
 
 Unit is able to be instantiated and bound in top level RTL.
+The design must compile and elaborate cleanly without errors.
 The unit must not break top level functionality such as propagating X through TL-UL interface, continuously asserting the interrupts, or creating undesired TL-UL transactions.
 
-### MEM_INSTANCED_80
+### PHYSICAL_MACROS_DEFINED_80
 
-80% of expected memories instantiated, using behavioral RAMs.
+All expected memories identified, representative macros instantiated.
```

Comment:
```suggestion
All expected memories identified, representative macros instantiated.
```

It it sufficient to use the appropriate primitives, e.g. `prim_ram_1p`, or does "macros instantiated" mean I should coordinate with e.g. Michael to get a customized SRAM macro from the memory generator?

Reply:
The former. Putting down `prim_ram_1p` in this example is sufficient.